src/redisraft.c

```diff
@@ -1800,9 +1800,14 @@ static void handleInfo(RedisModuleInfoCtx *ctx, int for_crash_report)
     RedisModule_InfoAddFieldULongLong(ctx, "fsync_avg_microseconds", avg);
 
     RedisModule_InfoAddSection(ctx, "snapshot");
-    RedisModule_InfoAddFieldCString(ctx, "snapshot_in_progress", rr->snapshot_in_progress ? "yes" : "no");
-    RedisModule_InfoAddFieldULongLong(ctx, "snapshots_loaded", rr->snapshots_loaded);
+    RedisModule_InfoAddFieldCString(ctx, "snapshot_filename", rr->config.rdb_filename);
+    RedisModule_InfoAddFieldULongLong(ctx, "snapshot_last_idx", rr->raft ? raft_get_snapshot_last_idx(rr->raft) : 0);
+    RedisModule_InfoAddFieldULongLong(ctx, "snapshot_last_term", rr->raft ? raft_get_snapshot_last_term(rr->raft) : 0);
+    RedisModule_InfoAddFieldULongLong(ctx, "snapshot_size", rr->outgoing_snapshot_file.len);
+    RedisModule_InfoAddFieldULongLong(ctx, "snapshot_time_secs", rr->last_snapshot_time);
```

Comment:
If I'm not mistaken, this is in `milliseconds` but we say `secs`.

Reply:
You're totally right - fixing